dd-bridge-android/src/main/kotlin/com/datadog/android/bridge/internal/BridgeSdk.kt

```diff
@@ -71,13 +72,22 @@ internal class BridgeSdk(
         if (configuration.sampleRate != null) {
             configBuilder.sampleRumSessions(configuration.sampleRate.toFloat())
         }
+
         if (configuration.site.equals("US", ignoreCase = true)) {
             configBuilder.useUSEndpoints()
         } else if (configuration.site.equals("EU", ignoreCase = true)) {
             configBuilder.useEUEndpoints()
         } else if (configuration.site.equals("GOV", ignoreCase = true)) {
             configBuilder.useGovEndpoints()
         }
+
+        val viewTracking = configuration.additionalConfig?.get(NATIVE_VIEW_TRACKING) as? Boolean
```

Comment:
probably it is better to use `.remove` here, unless we want this attribute to be propagated to the native level.

Reply:
Actually we can't as it's a non mutable map